index.bs

```diff
@@ -27,6 +28,8 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
 spec: storage; urlPrefix: https://storage.spec.whatwg.org/
     type: dfn
         text: storage shelf; url: storage-shelf
+        text: obtain a local storage shelf; url: obtain-a-local-storage-shelf
```

Comment:
we should probably consider exporting the definitions in the storage spec that need to be referenced?

Reply:
Agreed, as an end-state. I've reworked things a bit, which leaves references to "storage bucket" and "storage bottle", neither of which are exported. Those are in prose, and it seems awkward to drop them but wording suggestions welcome. 

"obtain a local storage bottle map" is exported, and that's the one that's used in an algorithm.